docs/platform/porting/system.md

```diff
@@ -2157,3 +2157,386 @@ $ sensorctl test accelerometer 100 1000 /* enable accelerometer with interval 10
 $ sensorctl test accelerometer 100 1000 0 /* enable accelerometer with interval 100 ms, 1s batch latency and always on option */
 $ sensorctl info accelerometer /* retrieve accelerometer sensor information */
 ```
+
+## Crash Framework
+
+### crash-manager
+
+crash-manager is a tool used to create crash reports when an application gets crashed.
+crash-manager is not usually invoked by manually, it is invoked automatically by the kernel when the crash occurs. Crash-manager generates the crash report and attaches the kernel-provided core, if requested. Crash-worker package provides the `99-crash-manager.conf` file, which configures kernel parameters for crash-manager to get invoked automatically on crash. For more information, see [`99-crash-manager.conf`](#99-crash-managerconf).
```

Comment:
Global: Please incorporate the comments provided earlier (https://github.com/Samsung/tizen-docs/pull/791#discussion_r312004276):

crash-manager is not invoked manually, it is invoked automatically by the kernel when the crash occurs. Crash-manager generates the crash report and attaches the kernel-provided core, if requested. The crash-worker package provides the `99-crash-manager.conf` file, which configures the kernel parameters for crash-manager to get invoked automatically on crash. For more information, see [`99-crash-manager.conf`](#99-crash-managerconf).